BUILDING.md

```diff
@@ -228,28 +227,77 @@ testing. Example:
 TAG=main docker-compose up
 ```
 
+## Nomad
+Grapl is transitioning to Nomad as our container orchestration. This will replace both AWS Fargate and docker-compose.
+
+### Nomad local development setup
+Have docker and docker-compose installed. Install Nomad and Consult following the instructions at 
+https://www.nomadproject.io/downloads and https://www.consul.io/downloads respectively.
+
+Install CNI plugins:
+
+```bash
+sudo mkdir -p /opt/cni/bin
+cd /opt/cni/bin
+sudo wget https://github.com/containernetworking/plugins/releases/download/v1.0.0/cni-plugins-linux-amd64-v1.0.0.tgz
+sudo tar -xf cni-plugins-linux-amd64-v1.0.0.tgz
+```
+
+If you are on ChromeOS you will also need to run the following due to https://github.com/hashicorp/nomad/issues/10902.
+This will allow Nomad to run bridge networks
+```bash
+sudo mkdir -p /lib/modules/$(uname -r)/
+sudo echo '_/bridge.ko' > /lib/modules/$(uname -r)/modules.builtin
+```
+
+### Running Nomad locally
```

Comment:
we definitely gotta improve this process at some point :)

Reply:
Yep. Once we've gotten this working properly we should be able to wrap it in a make call